geospatial/dted.ksy

```diff
@@ -0,0 +1,399 @@
+        repeat: expr
+        repeat-expr: 9
+      - id: reserved_for_nima_use_only2
+        type: str
+        size: 18
+        encoding: ASCII
+      - id: reserved_for_future_use3
+        type: str
+        size: 69
+        encoding: ASCII
+  acc_subregion:
+    seq:
+      - id: absolute_horizontal_accuracy_or_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: absolute_vertical_accuracy_or_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: relative_horizontal_accuracy_of_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: relative_vertical_accuracy_of_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: number_of_coordinates_in_accuracy_subregion_outline
+        type: str
+        size: 2
+        encoding: ASCII
+      - id: coordinate_pair
+        type: coordinate_pair_ddmmss_sh
+        repeat: expr
+        repeat-expr: 14
+  data_record:
+    seq:
+      - id: recognition_sentinal
+        type: u1
+      - id: data_block_count
+        type: b24
+      - id: logitude_count
+        type: u2
+      - id: latitude_count
+        type: u2
+      - id: elevations
+        type: data_record_elevation
+        repeat: expr
+        repeat-expr: _parent.uhl.number_of_latitude_points.value
+      - id: checksum
+        type: s4
+  data_record_elevation:
+    seq:
+      - id: is_negative
+        type: b1
+      - id: value
+        type: b15
+    instances:
+      meters:
+        value: value * (is_negative? -1 : 1)
+  ascii_int_ssss:
+    seq:
+      - id: text
+        type: str
+        size: 4
+        encoding: ASCII
+    instances:
+      value:
+        value: text.to_i
+  coordinate_pair_ddmmss_sh:
+    seq:
+      - id: latitude
+        type: angle_ddmmss_sh
+      - id: longitude
+        type: angle_dddmmss_sh
+  coordinate_pair_ddmmssh:
+    seq:
+      - id: latitude
+        type: angle_ddmmssh
+      - id: longitude
+        type: angle_dddmmssh
+  angle_ddmmss_sh:
+    seq:
+      - id: degrees
+        type: str
+        size: 2
+        encoding: ASCII
+      - id: minutes
+        type: str
+        size: 2
+        encoding: ASCII
+      - id: seconds
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: hemisphere
+        type: str
+        size: 1
+        encoding: ASCII
+    instances:
+      degree_decimal:
+        value: (degrees.to_i + minutes.to_i/60.0 + seconds.to_i/3600.0) * (hemisphere == 'N'? 1 : -1)
+  angle_dddmmss_sh:
+    seq:
+      - id: degrees
+        type: str
+        size: 3
+        encoding: ASCII
+      - id: minutes
+        type: str
+        size: 2
+        encoding: ASCII
+      - id: seconds
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: hemisphere
+        type: str
+        size: 1
+        encoding: ASCII
+    instances:
+      degree_decimal:
```

Comment:
just `value`?

Reply:
The purpose of this type is to do the conversion between the ASCII representation and Degree Decimal. It's a convenience thing.  This is because any elevation attribute is meaningless unless it is programmatically calculated what the latitude and longitude of that elevation point is.

Reply:
I mean if that `instance` should be named just `value`

Reply:
Done.  I'll add a doc tag which specifies that it is in the degree decimal format.